packages/typescript-estree/tests/ast-alignment/utils.ts

```diff
@@ -295,6 +295,27 @@ export function preprocessBabylonAST(ast: File): any {
           delete node.loc.start.index;
         }
       },
+      TSTypeQuery(node: TSTypeQuery) {
```

Comment:
can you add an comment with explanation why we need this and if there is babel ticket please link it here,
(its easier to maintain this that way)